src/jit/lsra.cpp

```diff
@@ -1962,6 +1966,8 @@ bool LinearScan::isRegCandidate(LclVarDsc* varDsc)
 
 void LinearScan::identifyCandidates()
 {
+    // Initialize the set of lclVars that are candidates for register allocation.
+    VarSetOps::AssignNoCopy(compiler, registerCandidateVars, VarSetOps::MakeEmpty(compiler));
 
```

Comment:
Do this, and the other two bitsets below (resolutionCandidateVars, splitOrSpilledVars) need to be initialized even if `lvaCount ==0` (condition below)?